src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -284,6 +284,14 @@
     type: command
     short-summary: Update a server.
     """
+helps['sql server dns-alias'] = """
+    type: group
+    short-summary: Manage a server's DNS aliases.
+    """
+helps['sql server dns-alias set'] = """
```

Comment:
Do you plan on having additional dns-alias commands?  If not, you should consider the command `az sql server set-dns-alias` instead.  We try to avoid subgroups with only one command unless you know it will be growing.

Reply:
We have other commands, just auto-generated help is good for them, so I didn't create ones. We are supporting create, delete, list, set and show